signal_test.go

```diff
@@ -0,0 +1,490 @@
+package dbus_test
```

Comment:
After changing the package and removing github.com/godbus/dbus from imports I'm getting this:

    [CoreOS-dev dbus]$ go test -v
    # github.com/godbus/dbus
    import cycle not allowed in test
    package github.com/godbus/dbus (test)
    	imports github.com/godbus/dbus/introspect
    	imports github.com/godbus/dbus
    
    FAIL	github.com/godbus/dbus [setup failed]

I'm inclined to leave it as is. Maybe with camelCased package name. Or maybe it should be `package introspect` (though the test doesn't have too much to do with this package).